containers/alpine/.devcontainer/devcontainer.json

```diff
@@ -5,13 +5,19 @@
 		// Update 'VARIANT' to pick an Alpine version: 3.13, 3.14, 3.15
 		"args": { "VARIANT": "3.15" }
 	},
-	
-	// Set *default* container specific settings.json values on container create. 
-	"settings": {},
 
-	// Add the IDs of extensions you want installed when the container is created.
-	// Note that some extensions may not work in Alpine Linux. See https://aka.ms/vscode-remote/linux.
-	"extensions": [],
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
```

Comment:
Since "extensions" and "settings" are empty: Should we omit "vscode" (and "customizations")? There is a discoverability argument against the removal, but at the same time this approach won't scale to more tools.

Reply:
This is a good point. Looking at several of the other devcontainer.json's in this repo, they all seem to contain `extensions` and `settings`, even if they're empty. But, this may have been because we considered these standard properties that everyone may want to use, which is less the case now (they're editor-specific). 

I think it makes sense to remove them from configurations where they're empty. In this next pass, I'll do that, plus the remaining `customizations` work for devcontainer.json's where they aren't empty.